tests/graphql_spec.rs

```diff
@@ -335,10 +336,13 @@ async fn test_execution() -> std::io::Result<()> {
                 let chrono_cache = init_in_memory_cache();
                 let server_ctx = AppContext::new(
                     blueprint,
-                    h_client,
-                    h2_client,
-                    init_env(),
-                    Arc::new(chrono_cache),
+                    TargetRuntime {
+                        http: h_client,
+                        http2_only: h2_client,
+                        env: init_env(),
+                        cache: Arc::new(chrono_cache),
+                        file: init_file(),
+                    },
```

Comment:
Can't we use `init_runtime` here?

Reply:
Fixed. I thought `h_client` and `h2_client` had some special config there, but nope.